tests/functional/Parser/Fixtures/Assignments/comment_multiline.php

```diff
@@ -1,5 +1,12 @@
 <?php declare(strict_types=1);
+
+use Helmich\TypoScriptParser\Parser\AST\MultilineComment;
+
 return [
+    new MultilineComment('/*
+Hello
+World
+*/', 1),
```

Comment:
I think a few more test cases with multi- and single line comments in various locations might be beneficial, here.

Reply:
Add some more places with single and multi line comments. The indentation gets lost for multiline comments. But i thinks this is fine.